### PR TITLE
Add coveralls test coverage

### LINKED ISSUE #18: Integrate Coveralls
Now that we have unit tests (#17), we could add [coveralls](https://coveralls.io/) or one of its alternatives to visualize and check our test coverage.

### PR DESCRIPTION
**Pull request by katkad**
This fixes #18

### CODE REVIEW

**Comment by lkiesow** [comment]
@katkad, thanks for the patch. I'll take care of the review as soon as #17 is merged.

File: .coveragerc [hunk 1]
```diff
@@ -0,0 +1,3 @@
+[report]
+omit =
+    */tests/*
```
Comment (lkiesow) [inline .coveragerc]: I don't think there should be a line break in this, should it?

File: .travis.yml [hunk 2]
```diff
@@ -21,6 +22,7 @@ after_success:
   - pip install sphinx sphinx-rtd-theme
   - make -C docs clean html
   - touch docs/build/html/.nojekyll # create this file to prevent Github's Jekyll processing
+  - coveralls
```
Comment (lkiesow) [inline .travis.yml]: Maybe move this into `after_success` since it isn't really a test?

```suggestion

after_success:
  - coveralls
```

**Comment by lkiesow** [comment]
@katkad, are you still working on this?

File: .travis.yml (hunk 2 above)
Comment (katkad) [reply .travis.yml]: Hello. I don't see test section here. It is in after_success section already.

File: .coveragerc (hunk 1 above)
Comment (katkad) [reply .coveragerc]: omit is multistring parameter. there are line breaks in example configuration: https://coverage.readthedocs.io/en/v4.5.x/source.html#execution

**Comment by katkad** [comment]
Hello, I updated my branch to your master. Is it okay now?

**Comment by lkiesow** [comment]
Thanks for the update. Merged.

**Pull request merged**